insalan/user/models.py

```diff
@@ -196,5 +196,18 @@ def send_password_reset(user_object: User):
             fail_silently=False,
         )
 
+    @staticmethod
+    def send_kick_mail(user_object: User, team_name: str, user_name: str):
+        """
+        Send a mail to a user that has been kicked.
+        """
+        send_mail(
+            _("Vous avez été exclu.e de votre équipe"),
+            _("Vous avez été exclu.e de l'équipe ") + team_name
+            + _(" par ") + user_name + _("."),
```

Comment:
do we... really need to translate a period? Perhaps this whole thing should be a template like `_("Vous avez été exclu⋅e de l'équipe %s par %s.") % (team_name, user_name)` ? I think that's how that works.

Reply:
Yep, seems better :+1:

Reply:
Btw do we really want to expose to an user who kicked them? Social interactions are probably hard enough if a player needs to be kicked, to need to make things worse.